winui-toc.html

```diff
@@ -104,6 +104,16 @@
             <li><a href="/winui/Barcode/barcode-customization">Customization</a></li>
         </ul>
     </li>
+    <li>
+        BusyIndicator
+        <ul>
+            <li><a href="/winui/BusyIndicator/Overview">Overview</a></li>
+            <li><a href="/winui/BusyIndicator/Getting-Started">Getting Started</a></li>
+            <li><a href="/winui/BusyIndicator/AnimationType">AnimationType</a></li>
```

Comment:
@Priyanka-Vijayasankar Provide space for AnimationType

Reply:
Provided space in the Animation Type in next PR.